### DISCUSSION TITLE
did something change related to Tags?

### DISCUSSION THREAD

**Discussion by mnoreen** [gitextensions/gitextensions#9533] [Q&A]
For some reason, we are no longer seeing new "Tags" in our repo/branches.  Did something recently change related to tags?  It used to be I could create a tag (say, just after a release and we would create a lightweight tag "Released_X" and then push that branch and include the tag.  Then, when others on the team pulled that branch, they would get the tag (along with the latest commits).  

Now, we don't seem to be seeing new tags at all.  

So, as a test... I created tag.  From a bash window, I entered "git push --tags" and if I then enter "git tag", I see said tag.  But... my team member does a pull and still does not see the tag.

**Comment by gerhardol** [comment]
GE has not changed the behaviour, not in Git what I know.
Most tags have to be explicitly fetched, see GE fetch/pull dialoge

Can be configured in Git too https://git-scm.com/docs/git-config#Documentation/git-config.txt-remoteltnamegttagOpt
